.gitignore

```diff
@@ -1,8 +1,8 @@
-.vs/
-obj/
-bin/
-artifacts/
-wwwroot/
-node_modules/
-*.user
+.vs/
+obj/
+bin/
+artifacts/
+wwwroot/
+node_modules/
+*.user
 .pnpm-debug.log
```

Comment:
I know you didn't break it, but the _.gitignore_ file [should end with an empty line](https://lombiq.atlassian.net/wiki/spaces/DEV/pages/19496967/Creating+a+new+Visual+Studio+project+or+solution#Creating-a-new-project).

I've fixed that now.

Reply:
Thanks and agreed!